scripts/prefab_descriptors/roseglasseshat.lua

```diff
@@ -0,0 +1,26 @@
+local function Describe(self, context)
```

Comment:
Last thing is for you to cede this file by putting the Insight copyright at the top - you can copy it from pretty much any other file that has it.